Functions/Standard_deviation.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+import Power_Function as pf
+
+
+def standard_deviation(input):
+    count = len(input)
+    if count > 0:
+        # if the length of argument list is 1, return 0 directly
+        if count == 1:
+            return 0;
+        # compute mean
+        args = np.array(input)
+        mean, s = total_count(args)/count, 0
+        # get a new list after each item minus mean, and square
+        args_new = args - mean
+        for i in range(count):
+            args_new[i] = pf.power_integer(args_new[i], 2)
+
+        return pf.power_fractional(total_count(args_new) / count, 0.5)
```

Comment:
I changed the power function, so you cannot call "power_fractional" directly. Calling "power_function" instead will give you a more precise answer, and it will have the same efficiency.

Reply:
fixed